docs/CI-SETUP.md

```
@@ -0,0 +1,147 @@
+
+### Release
+| Tool | Role |
+|---|---|
+| [Changesets](https://github.com/changesets/changesets) | Versioning + changelog + npm publish (with provenance). |
+| [npm-run-all2](https://github.com/bcomnes/npm-run-all2) | `run-p` / `run-s` for parallel/sequential script orchestration. |
+
+## npm scripts (gate of truth)
+
+The local equivalent of CI is `npm run verify`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)